mathesar_ui/src/stores/abstract-types/type-configs/email.ts

```diff
@@ -0,0 +1,12 @@
+import { faEnvelope } from '@fortawesome/free-solid-svg-icons';
+import type { AbstractTypeConfiguration } from '../types';
+
+const emailType: AbstractTypeConfiguration = {
+  icon: { data: faEnvelope },
+  defaultDbType: 'mathesar_types.email',
+  cell: {
+    type: 'email',
```

Comment:
`type` here can just be `string`, with `multiLine: false`. We do not have to define a dedicated `email` type since we aren't doing anything additionally as of the moment.

Reply:
Thanks @pavish. This should be resolved by 43446c7768b33d027db1751a8dd49760020647dd.